pkgs/applications/science/electronics/kicad/update.sh

```diff
@@ -175,6 +175,13 @@ done
 printf "}\n"
 } > "${tmp}"
 
+grep "\"\"" "${tmp}"
```

Comment:
I guess, it would be more robust if the script was written assuming `set -e`. But this is a big script now...

Reply:
i'm fairly unfamiliar with `set -e`, looks like that wouldn't convey that the error is most likely caused by running out of space

Reply:
Of course, `set -e` would not know that there is disk involved, so it won't provide informative error messages by itself. The point of using it is that shell scripts fail for other reasons and they better be failing immediately whenever something unexpected occurs.

Reply:
i've added `set -e`, waiting for a rebuild on latest revs and got to go afk for a while, maybe don't merge this until then